src/main/java/com/flagsmith/FlagsmithClient.java

```diff
@@ -105,7 +108,9 @@ public Flags getIdentityFlags(String identifier)
    */
   public Flags getIdentityFlags(String identifier, Map<String, Object> traits)
       throws FlagsmithClientError {
-    if (flagsmithSdk.getConfig().getEnableLocalEvaluation()) {
+    FlagsmithConfig config = getConfig();
+    Boolean useEnvironmentDocument = config.getEnableLocalEvaluation() | config.getOfflineMode();
+    if (useEnvironmentDocument) {
```

Comment:
Arguably, this calls for defining `FlagsmithConfig.getShouldUseEnvironmentDocument`.

Doesn't look like a hard blocker for me, though.

Reply:
Yep, good point, I've done that.